.github/workflows/pr-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Plugins Platform PR - CI
```

Comment:
Does this only run e2e tests or also the rest of the CI checks? (unit tests, lint, etc)

Reply:
Yep it even runs Trufflehog and Spellcheck https://github.com/grafana/logs-drilldown/actions/runs/14734763800/job/41357962193?pr=1215
![Screenshot 2025-04-30 at 9 07 11 AM](https://github.com/user-attachments/assets/c84bf1b9-d06c-4420-908c-12b09919dd30)